docs/accessibility.md

```diff
@@ -61,9 +60,9 @@ const App = () => {
 
 ```js
 import AnnounceDecorator from '@enact/ui/AnnounceDecorator';
-import React from 'react';
+import {Component} from 'react';
```

Comment:
Shouldn't we fix the line 45 and moonstone?

Reply:
Good catch for line 45. It must be undone.
The rest of the stuff about moonstone will be taken care of in PLAT-139104.

Reply:
I can found another 7 `import React from 'react';` in this branch